src/ucp/api/ucp.h

```diff
@@ -941,6 +953,30 @@ typedef struct ucp_listener_attr {
 } ucp_listener_attr_t;
 
 
+/**
+ * @ingroup UCP_WORKER
+ * @brief UCP listener's connection request attributes.
+ *
+ * The structure defines the attributes which characterize
+ * the particular connection request received on the server side.
+ */
+typedef struct ucp_conn_request_attr {
+    /**
+     * Mask of valid fields in this structure, using bits from
+     * @ref ucp_conn_request_attr_field.
+     * Fields not specified in this mask will be ignored.
+     * Provides ABI compatibility with respect to adding new fields.
+     */
+    uint64_t                field_mask;
+
+    /**
+     * The address of the remote client who sent the connection request to the
+     * server.
+     */
+    ucs_sock_addr_t         client_address;
```

Comment:
should be sockaddr_storage
like in ucp_listener_attr_t

Reply:
ok so will change it to sockaddr_storage in uct as well, yes?

Reply:
no because in UCT it's arguments to a function, in UCP it's filled by the function